deploy/docker-compose.prysm.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
+
+volumes:
+  prysm:
+    driver: local
+
+services:
+  prysm:
+    image: gcr.io/prysmaticlabs/prysm/beacon-chain:v2.0.2
+    restart: always
+    ports:
+      - 4000:4000
+    command:
+      - --datadir=/data
+      - --rpc-host=0.0.0.0
+      - --monitoring-host=0.0.0.0
+      - --http-web3provider=http://geth:8545
+      - --accept-terms-of-use
```

Comment:
Lets add --slots-per-archive-point=64 here. See: https://github.com/prysmaticlabs/prysm/issues/9724

We must have the same flag for the pyrm helm chart in case it's deployed for the oracle